src/matrix.js

```diff
@@ -802,7 +802,8 @@ export class AbstractMatrix {
     return diag;
   }
 
-  norm(type = 'frobenius') {
+  norm(type) {
+    if (type === undefined) type = 'frobenius';
```

Comment:
you can skip this assignation here just adding this as a condition on the line 810, something like

```ts
} else if (type === 'frobenius' || type === undefined) {
```

Reply:
I did it this way on purpose to be consistent with e.g. `eye`. In my own code I would probably write the whole method as:

```
norm(type) {
  switch(type ?? 'frobenius') {
    case 'max':
      return this.max();
    case 'frobenius':
      return Math.sqrt(this.dot(this));
    default:
      throw new RangeError(`unknown norm type: ${type}`);
}
```

Reply:
I personally like more your answer over the parameters assignation